File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0:
            return False
        
        l = list(str(x))

        for i in range(len(l)//2):
            if l[i] != l[-1*i - 1]:
                return False
        
        return True

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if (x < 0){
            return false;
        }

        String xStr = Integer.toString(x);
        StringBuilder sb = new StringBuilder(xStr);
        sb.reverse();

        if (xStr.equals(sb.toString())){
            return true;
        } else{
            return false;
        }
    }
}